src/common/xo-week-heatmap/index.js

```diff
@@ -44,66 +67,66 @@ export default class XoWeekHeatmap extends Component {
   }
 
   componentWillReceiveProps (nextProps) {
-    this._updateState(nextProps)
+    this._updateDays(nextProps)
   }
 
   componentWillMount () {
-    this._updateState()
+    this._updateDays()
   }
 
-  _computeColorGen = data => scaleSequential(interpolateViridis)
-    .domain(extent(data, value => value.value).reverse())
-
-  _updateState (props = this.props) {
+  _updateDays (props = this.props) {
     const {
       data,
       intl: {
         formatTime
       }
     } = props
-    const colorGen = this._computeColorGen(props.data)
-    const matrix = {}
+    const days = {}
```

Comment:
Make it an array containing the number of days between the first and last record.